### ISSUE TITLE
ipa_installer determines wrong Executable

### ISSUE THREAD

**Issue by ay-kay** [dpnishant/appmon#19]
The `getMachOExecutable()` function in `ipa_installer.py` is not working properly. 

In my case the `file` command returned `Mach-O 64-bit executable arm64` for the executable file name. In consequence, none of the files matched the expected string `Mach-O universal binary` and therefore the last value remaining in `output` (the last file) was assumed to be the executable (which was in fact a ASCII file in my case). 

In general, it would be much more precise to determine the executable by reading the `CFBundleExecutable` property from the `Info.plist` file.

Brief attempt (without error handling):

```python
import plistlib

def getMachOExecutable(app_path):
    plist_path = os.path.join(app_path, "Info.plist")
    plist = plistlib.readPlist(plist_path)
    executable = plist["CFBundleExecutable"]
    return os.path.join(app_path, executable)
```

**Comment by dpnishant** [comment]
Hi @ay-kay,

If the app is not private, can you please let me know its name or send me the IPA to reproduce this issue?

**Comment by dpnishant** [comment]
Fixed with f39ef41. Please reopen if issue still persists
